tools/ocean/argo_decoder.xml

```diff
@@ -0,0 +1,63 @@
+<tool id="coriolis_data_processing_chain_for_argo_floats" name="coriolis data processing" version="@VERSION@">
+    <description>chain for argo floats</description>
+    <macros>
+        <token name="@VERSION@">061a</token>
+        <token name="@VERSION_DOCKER@">@VERSION@-galaxy</token>
+    </macros>
+    <requirements>
+        <container type="docker">ghcr.io/euroargodev/coriolis-data-processing-chain-for-argo-floats:@VERSION_DOCKER@</container>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        
+        ## prepare outputs directory
+        mkdir -p /mnt/data/output/log &&
```

Comment:
why do we need /mnt at all? Is this something the APP needs to?

Reply:
I think the decoder need this tree structure for now. But that's a question more for @leobr29

Reply:
ok!